config/config.go

```diff
@@ -39,3 +41,96 @@ func DetermineUseLiveData(logger golog.Logger, liveData *bool, sensors []string)
 	}
 	return useLiveData, nil
 }
+
+// AttrConfig describes how to configure the service.
+type AttrConfig struct {
+	Sensors             []string          `json:"sensors"`
+	ConfigParams        map[string]string `json:"config_params"`
+	DataDirectory       string            `json:"data_dir"`
+	UseLiveData         *bool             `json:"use_live_data"`
+	DataRateMs          int               `json:"data_rate_msec"`
+	MapRateSec          *int              `json:"map_rate_sec"`
```

Comment:
[nit] Let's change `DataRateMs` to reflect how it's defined in JSON and how `MapRateSec` is implemented

```suggestion
	DataRateMsec          int               `json:"data_rate_msec"`
	MapRateSec          *int              `json:"map_rate_sec"`
```